detectable/src/main/java/com/synopsys/integration/detectable/detectables/bitbake/parse/BitbakeEnvironmentParser.java

```diff
@@ -13,12 +13,14 @@
 public class BitbakeEnvironmentParser {
     private static final String ARCHITECTURE_VARIABLE_NAME = "MACHINE_ARCH";
     private static final String LICENSES_DIR_VARIABLE_NAME = "LICENSE_DIRECTORY";
+    private static final String MACHINE_VARIABLE_NAME      = "MACHINE";
     private static final String QUOTE_CHARS = "\"'";
     private final Logger logger = LoggerFactory.getLogger(this.getClass());
     private final Predicate<String> isArchitectureLine = l -> l.startsWith(ARCHITECTURE_VARIABLE_NAME + "=");
     private final Predicate<String> isLicensesDirLine = l -> l.startsWith(LICENSES_DIR_VARIABLE_NAME + "=");
+    private final Predicate<String> isMachineLine = l -> l.startsWith(MACHINE_VARIABLE_NAME + "=");
 
-    public BitbakeEnvironment parseArchitecture(List<String> bitbakeEnvironmentCmdOutput) {
+    public BitbakeEnvironment parse(List<String> bitbakeEnvironmentCmdOutput) {
```

Comment:
Even prior to this change this method was parsing more than just architecture. So, I renamed it to be more general.